### PR TITLE
[#173703056] Fix accessibility on Fiscal code

### PR DESCRIPTION
**Pull request by CrisTofani**
**Short description:**
This PR implements the accessibility on the fiscal code component and screen

### CODE REVIEW

File: ts/components/FiscalCodeComponent.tsx [hunk 1]
```diff
@@ -449,7 +452,29 @@ export default class FiscalCodeComponent extends React.Component<Props> {
 
   public render(): React.ReactNode {
     return (
-      <View>
+      <View
+        accessible={true}
+        accessibilityLabel={
```
Comment (Undermaken) [inline ts/components/FiscalCodeComponent.tsx:457]: lgtm
To improve _render_ readability can could we refactor as follows?

```suggestion
        accessibilityLabel={this.accessibilityText.accessibilityLabel}
        accessibilityHint={this.accessibilityText.accessibilityHint}
```

```typescript
get accessibilityText(): Record<
    "accessibilityLabel" | "accessibilityHint",
    string
  > {
    if (this.props.type === "Preview") {
      return {
        accessibilityLabel: I18n.t(
          "profile.fiscalCode.accessibility.preview.label"
        ),
        accessibilityHint: I18n.t(
          "profile.fiscalCode.accessibility.preview.hint"
        )
      };
    }
    const isLandScape = this.props.type === "Landscape";
    if (this.props.getBackSide) {
      return {
        accessibilityLabel: I18n.t(
          "profile.fiscalCode.accessibility.rear.label"
        ),
        accessibilityHint: isLandScape
          ? ""
          : I18n.t("profile.fiscalCode.accessibility.rear.hint")
      };
    }
    // goBackSide === false
    return {
      accessibilityLabel: I18n.t(
        "profile.fiscalCode.accessibility.front.label",
        {
          code: this.props.profile.fiscal_code,
          name: this.props.profile.name,
          family_name: this.props.profile.family_name
        }
      ),
      accessibilityHint: isLandScape
        ? ""
        : I18n.t("profile.fiscalCode.accessibility.front.hint")
    };
  }
```

File: ts/components/FiscalCodeComponent.tsx [hunk 2]
```diff
@@ -334,6 +334,9 @@ export default class FiscalCodeComponent extends React.Component<Props> {
             : [styles.fullText, fullStyle]
         ]}
         selectable={selectable}
+        accessible={true}
```
Comment (Undermaken) [inline ts/components/FiscalCodeComponent.tsx:337]: I guess we need to add
`accessibilityRole={"button"}`
in `ts/screens/profile/ProfileMainScreen.tsx` line 534 to read "button" so the user can know he can access it by double tap

File: locales/en/index.yml [hunk 3]
```diff
@@ -127,6 +127,16 @@ inbox:
   enableCallToActionDescription: Enable the inbox to view messages
 profile:
   fiscalCode:
+    accessibility:
+      preview:
+        label: Preview of your fiscal code document
+        hint: Activate to show the facsimile of your fiscal code document
+      front:
+        label: Front-side of the document code {{code}} of {{name}} {{family_name}}
```
Comment (Undermaken) [inline locales/en/index.yml:135]: Could we add all other informations?
- place of birth
- sex
- province
- date of birth

File: locales/it/index.yml [hunk 4]
```diff
@@ -127,6 +127,16 @@ inbox:
   enableCallToActionDescription: Abilita l'inbox per visualizzare i messaggi
 profile:
   fiscalCode:
+    accessibility:
+      preview:
+        label: Anteprima del tuo codice fiscale
+        hint: Attiva per visualizzare il facsimile del tuo codice fiscale
+      front:
+        label: Parte anteriore del documento codice fiscale {{code}} di {{name}} {{family_name}}
+        hint: Attiva per passare alla visualizzazione landscape
```
Comment (Undermaken) [inline locales/it/index.yml:136]:
```suggestion
        hint: Attiva per passare alla visualizzazione a schermo intero
```

File: ts/components/FiscalCodeComponent.tsx (hunk 2 above)
Comment (CrisTofani) [reply ts/components/FiscalCodeComponent.tsx:337]: @Undermaken should we consider it a "button" even if it's not properly a button? 
I have chosen to explain it by the hint `profile.fiscalcode.accessibility.preview.hint`: _Activate to show the facsimile of your fiscal code document_

what do you think?

File: locales/en/index.yml (hunk 3 above)
Comment (CrisTofani) [reply locales/en/index.yml]: added in ca541de

File: locales/it/index.yml (hunk 4 above)
Comment (CrisTofani) [reply locales/it/index.yml]: fixed in ca541de

File: ts/components/FiscalCodeComponent.tsx (hunk 1 above)
Comment (CrisTofani) [reply ts/components/FiscalCodeComponent.tsx]: added in ca541de

File: ts/components/FiscalCodeComponent.tsx (hunk 2 above)
Comment (Undermaken) [reply ts/components/FiscalCodeComponent.tsx:337]: @CrisTofani I noted on iOS the hint is not pronounced when the accessibility label is defined. This is why I can't understand if that element could be activated 
Moreover the CF preview acts like a button (it is wrapped inside a TouchableDefaultOpacity )
https://github.com/pagopa/io-app/blob/836a9b1dd7159d1bb73e4ca9f17efc5258ceabc4/ts/screens/profile/ProfileMainScreen.tsx#L534

Comment (CrisTofani) [reply ts/components/FiscalCodeComponent.tsx:338]: fc19996 adds the role **button** on `preview` and `full` visualization and **none** on `landscape`

**Pull request merged**